### PR TITLE
Integrate Ethnicity fields with API

### PR DESCRIPTION
**Pull request by RamuKammagani-osi**

### CODE REVIEW

**Comment by edgibbs** [comment]
retest this please

**Comment by edgibbs** [comment]
retest this please

File: app/javascript/Clients/clientIdPage/RaceEthnicity/EthnicityDetails.js [hunk 1]
```diff
@@ -1,10 +1,10 @@
...
-  'Yes',
-  'No',
-  'Unknown',
-  'Abandoned',
-  'Declined to answer',
-]);
+const HISPANIC_LATINO_ORIGIN = Object.freeze(
+  // 'Yes',
+  // 'N',
+  'U'
```
Comment (edgibbs) [inline app/javascript/Clients/clientIdPage/RaceEthnicity/EthnicityDetails.js]: So HISPANIC LATINO ORIGIN can just be 'U'?  If so at least need to remove the commented out options.

File: lib/child_clients/child_client.rb [hunk 2]
```diff
@@ -29,5 +29,64 @@ class ChildClient < Dry::Struct
...
+    attribute :hispanic_origin_code, Types::String.optional
+    attribute :children_serv_indicator, Types::String.optional
+    attribute :currently_regional_centeer_indicator, Types::String.optional
+    attribute :currently_other_description, Types::String.optional
+    attribute :previously_received_indicator, Types::String.optional
+    attribute :previously_regional_center_indicator, Types::String.optional
+    attribute :previously_other_description, Types::String.optional
+    attribute :health_care_plan_indicator, Types::String.optional
+
+    # "limitation_on_scphealth_indicator" : false,
```
Comment (edgibbs) [inline lib/child_clients/child_client.rb]: Remove commented out attributes.

File: app/javascript/Clients/clientIdPage/RaceEthnicity/Ethnicity.js [hunk 3]
```diff
@@ -29,9 +91,11 @@ export default class Ethnicity extends React.Component {
       : this.setState({ ethnicityDetail: value.toString() });
   }
   render() {
+    console.log(this.state.latinoOrigin);
```
[the remark this answers is not in the document]
Comment (RamuKammagani-osi) [reply app/javascript/Clients/clientIdPage/RaceEthnicity/Ethnicity.js]: Yes that's right! I forgot to remove that

**Pull request merged**